lib/galaxy_test/base/populators.py

```diff
@@ -314,7 +314,7 @@ def __init__(self, dataset_populator, workflow_populator, history_id, workflow_i
         self.invocation_id = invocation_id
 
     def _output_name_to_object(self, output_name):
-        invocation_response = self.dataset_populator._get(f"workflows/{self.invocation_id}/invocations/{self.workflow_id}")
+        invocation_response = self.dataset_populator._get(f"workflows/{self.workflow_id}/invocations/{self.invocation_id}")
         api_asserts.assert_status_code_is(invocation_response, 200)
```

Comment:
We can just use the invocation ID now also.

Reply:
Good point, addressed in a commit that I'll push soon.